index.html

```diff
@@ -49,23 +53,41 @@ <h2>Autarky is</h2>
+            <h1>Can my municipality supply itself with electricity from sun and wind?</h1>
             <p>
                 This map visualises the possibility for renewable electricity autarky in Europe ---
-                for all municipalities, all regions, all countries, and all of Europe.
-                Possibility for autarky means <b>being able to fulfill local electricity demand with
-                100% renewable electricity from wind and solar power generated locally</b>.
+                for all municipalities, all regions, all countries, and all of Europe. 
+                No matter where you live, here you can find out whether enough electricity from 
+                sun and wind can be generated in your region.
+            </p>
+            <h2>But what does it mean to be autark / self-sufficient?</h2>
+            <p>   
+                Possibility for autarky means being able to fulfil local electricity demand with
+                100% renewable electricity from solar and wind power generated locally. 
+                It makes it possible for entire communities to produce their own electricity - for
+                example using solar systems on the roofs and community wind farms - and then consume it.
+            </p>
+            <h2>Can Europe be powered by sun and wind alone?</h2>
+            <p>
+                The potential of solar and wind energy is large enough to supply Europe with 100%
+                renewable electricity. In many cases at regional or local level, self-sufficiency is
+                possible using renewable electricity only. However, in particular densely populated
+                metropolis will not be able to supply themselves with 100% renewable electricity.
+                Surrounding regions can may cover their needs.
             </p>
+            <h2>What does the map show me?</h2>
             <p>
-                Areas will not be able to become autarkic when their local electricity demand exceeds
-                the potential of renewable electricity generation. The potential used here
-                is based on technical constraints and some social constraints; for example we do
-                not allow generation on environmentally protected areas and we also prohibit open
-                field solar electricity on arable land. We call this potential the "technical-socal-potential"
-                and you can find the exact definition in our peer-reviewed article, see below.
-                Technically, much more electricity can be generated, and thus we consider autarky
-                impossible only if local electricity demand exceeds a potential based solely on technical
-                constraints:
+                The map shows the potential for renewable electricity self-sufficiency in Europe for
+                individual countries, federal states and municipalities. The potential shown is based on
```

Comment:
`federal states` is not correct for all countries

Reply:
changes it to regions